src/deadline/client/ui/widgets/job_attachments_tab.py

```diff
@@ -308,6 +320,27 @@ def get_asset_references(self) -> AssetReferences:
         """
         return self.auto_detected_attachments.union(self.attachments)
 
+    def get_require_paths_exist(self) -> bool:
+        """
+        Returns the checkbox value of whether to allow empty paths or not.
+        """
+        return self.general_settings.require_paths_exist.isChecked()
+
+
+class JobAttachmentsGeneralWidget(QWidget):
+    """
+    A Widget that contains general settings for a specific submission.
+    """
+
+    def __init__(self, parent=None) -> None:
+        super().__init__(parent=parent)
+
+        layout = QHBoxLayout(self)
+        layout.setContentsMargins(0, 0, 0, 0)
+
+        self.require_paths_exist = QCheckBox("Require All Input Paths Exist", parent=self)
```

Comment:
We'd like to use sentence case for items like this, `"Require all input paths exist"`. We haven't gone through the whole UI to adjust this yet, but easy to apply this to new options.